docs/source/maintenance/restarting_node.rst

```diff
@@ -37,6 +37,9 @@ Although it can be a great idea for some of the cases, but please consider that
 	If blocks are less than should be – the option of reusing WSV will not work for you.
 	Please, restore it from blocks.
 
+If you want to reuse WSV state, start iroha with `--reuse_state` flag.
+Given this flag, iroha will not reset or overwrite the state database if it fails to start for whatever reason.
+
```

Comment:
I'd make a paragraph out of it, like:

Enabling WSV Reuse
^^^^^^^^^^^^^^^^^^

If you want to reuse WSV state, start Iroha with `--reuse_state` flag.
Given this flag, Iroha will not reset or overwrite the state database if it fails to start for whatever reason.

Reply:
done